### ISSUE TITLE
RFC: Splitting PHP formulas into CLI / Apache / FPM / CGI / SRC

### ISSUE THREAD

**Issue by ablyler** [Homebrew/homebrew-php#1999]
@Homebrew/php: I propose that we split the current php formulas into more specific ones.  This will allow us to have more bottles, and remove a lot of the options from the current formulas.  We are already moving this way by moving some of the built in php extensions (such as gmp / snmp / mcrypt) into their own formulas.

Another nice side effect of this is that the php*-src formula can be a `keg_only`.  This will allow extensions to be upgraded without the current `brew unlink phpVERSION1 && brew link phpVERSION2`.

I have a [work-in-progress branch](https://github.com/Homebrew/homebrew-php/compare/master...ablyler:php-the-great-split-for-bottles) that can be reviewed.

**Comment by webflo** [comment]
:+1: I would love to use multiple php versions build my homebrew with phpenv.

**Comment by dbergey** [comment]
Agreed: separate pieces == better

**Comment by Maks3w** [comment]
:+1:  This fix part of #1952

**Comment by HardieBoeve** [comment]
:+1:

**Comment by webflo** [comment]
@ablyler Thanks for working on. Could you summarize the current status? I would like to help/test/whatever.

**Comment by josegonzalez** [comment]
Closing as this appears to have stalled. The initial work on this is in #2122. If you'd like to help with that issue please do.
